build.sbt

```diff
@@ -34,6 +34,9 @@ val io = (project in file("io"))
       Vector(scalaCompiler.value % Test, scalaCheck % Test, scalatest.value % Test)
     } ++ Vector(swovalFiles),
     libraryDependencies ++= Seq(jna, jnaPlatform),
+
+    Test / fork := true,
```

Comment:
What was the motivation for this change? I find that scalatest seems to leak class loaders (at least compared to utest) so I don't mind forking, but was still curious.

Reply:
I was using cross JDK forking (https://developer.lightbend.com/blog/2018-07-02-sbt-1-2-0/index.html#cross-jdk-forking) locally:

```
> java++ 8!
> io/testOnly sbt.io.IOSpec
> java++ 11!
> io/testOnly sbt.io.IOSpec
```

Reply:
Got it. That makes sense.